programs/stylua.nix

```diff
@@ -159,10 +159,7 @@ let
 
       settings = filterOutEmptyAttrs (filterOutNull cfg.settings);
     in
-    if settings != { } then
-      configFormat.generate "stylua.toml" settings
-    else
-      null;
+    configFormat.generate "stylua.toml" settings;
```

Comment:
Is it standard to always generate a file regardless?
The intent was to avoid generating a config file at all unless actually needed.
I'm not sure if I've missed something.

Reply:
Yeah, Nix is lazily evaluated, so it would only generate the file if the branch is also evaluated.